File: schrottverwaltung/src/main/java/de/fhdortmund/schrottverwaltung/mitarbeiter/Fachbereich.java
package de.fhdortmund.schrottverwaltung.mitarbeiter;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "fachbereich")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Fachbereich {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long fachbereichNr;

    private String bezeichnung;
}

File: schrottverwaltung/src/main/kotlin/de/fhdortmund/schrottverwaltung/mitarbeiter/Fachbereich.kt
package de.fhdortmund.schrottverwaltung.mitarbeiter

import jakarta.persistence.*

@Entity
@Table(name = "fachbereich")
data class Fachbereich(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var fachbereichNr: Long,
    var bezeichnung: String,
)
